website/content/docs/upgrading/upgrade-to-1.16.x.mdx

```diff
@@ -81,6 +81,22 @@ userattr="userprincipalname"
 Refer to the [LDAP auth method (API)](/vault/api-docs/auth/ldap) page for
 more details on the configuration.
 
+### JWT auth login requires bound audiences on the role
+
+JWT auth roles of type "jwt" require the bound_audiences claim to match at
+least one of the JWT's aud claims. The JWT's aud claim can be a single string
+or a list of strings as per
+[RFC 7519 Section 4.1.3](https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.3).
+Prior to 1.16.3, the JWT auth method would ignore token aud claims that were
+not a list of strings.
+
+The consequence of not configuring this correctly means users may not be able
+to login to Vault. To fix the issue, update the role's `bound_audiences`
+parameter to match the `aud` claim on the JWT.
+
+Refer to the [JWT auth method (API)](/vault/api-docs/auth/jwt) page for
+more details on the configuration.
```

Comment:
```suggestion
<Warning title="JWT auth login requires bound audiences on the role">

  The bound_audience claim of "jwt" roles **must** match at least one of the
  associated `aud` claims. The `aud` claim can be a single string or a list of
  JSON web tokens that are
  [RFC 7519 Section 4.1.3](https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.3)
  compliant.

  Users may not be able to log into Vault if the JWT role is configured
  incorrectly. For additional details, refer to the
  [JWT auth method (API)](/vault/api-docs/auth/jwt) documentation.

</Warning>
```

Style correction: Write in active voice, avoid "this" as a pronoun, avoid possessives